int32  userID
int32  robotID     # -1 all, 0 none, 107 specific
int32  severity
string displayMsg  # more verbose message, to display on screen
string speakMsg    # if len(speakMsg) > 0, speak this string
